spec/models/graphql/analyzer_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+describe Graphql::Analyzer do
+  let(:user) { create(:user) }
+  let(:query_string) { %|{ user(id: "#{user.to_global_id}") { id } }| }
+  let(:analyzer) { Graphql::Analyzer.new(AppSchema) }
```

Comment:
Here is where we "wrap" the schema with a decorator that will add functionality that will return data relating to the queries that were made.